data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+					goto defww
+				`	(Go with the others and check the back gate.)`
+			`	Once you get to the back gate, you and the Kimek there stay quiet and try to listen for anyone approaching the fort. Sure enough, you hear scuttling Kimek legs swishing through the grass, as the attackers come circling around. Fortunately, not many came to this gate, so you and the Kimek don't struggle much to keep it closed. The group at the front gate isn't so lucky, however, and after a brief minute the whistle sounds again. Your group comes back to the courtyard to find that the gate has been busted open and that the attackers have won.`
+				goto lost
+			label defww
+			`	At first, holding the gate isn't too troublesome, but after a few minutes pass, it starts being pounded by some form of battering ram, and before long the group there can't hold it closed. At the same time, you hear a commotion from behind the shack, and as you and others go there to check, you arrive to see the small gate open. The attackers having made their way inside via both gates, the whistle sounds again, marking the end of the battle and the victory of the other side.`
+				goto lost
+			label att
+			`	He brings you to the nearby camp, where many Kimek, Arachi, and even a few Saryds are gathered for the event. Most of them are waiting in anticipation inside one of the tents, or looking up at the fort; all of them have long, blue strips of cloth over their own clothing. Once she notices you two, another Kimek organizer, donning what looks like a thin gambeson, comes to say hello. She greets her colleague, who introduces you and tells her you chose to join the attackers, before leaving the camp to welcome a family who has come to watch the event.`
+			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so," she pauses, looking at you as if just realizing you can't crawl up walls. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`
```

Comment:
```suggestion
			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`
```